cms/tests/frontend/integration/pagetree.js

```diff
@@ -1314,7 +1314,7 @@ casper.test.begin('Pagetree remembers which nodes are opened and which ones are
                     window.location.reload();
                 })
                 .waitUntilVisible('.cms-pagetree-jstree')
-                .wait(1000, function () {
+                .wait(2000, function () {
```

Comment:
not really :) they are very arbitrary and subject to change, i wouldn't touch it